snapcraft/internal/review_tools/_runner.py

```diff
@@ -0,0 +1,50 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright 2019 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import json
+import pathlib
+import subprocess
+
+from . import errors
+
+
+def is_available() -> bool:
+    review_tools_bin = pathlib.Path("/snap/bin/review-tools.snap-review")
+    return review_tools_bin.exists()
+
+
+def run(*, snap_filename: str) -> None:
+    review_tools_bin = pathlib.Path("/snap/bin/review-tools.snap-review")
+    if not review_tools_bin.exists():
+        raise errors.ReviewToolNotAvailableError()
```

Comment:
FWIW, I think the proper way to do this is to catch FileNotFoundError on check_output()

Reply:
In fact, do that because `/snap/bin` may not be in path as we have seen before...